festim/boundary_conditions/fluxes/mass_flux.py

```diff
@@ -0,0 +1,27 @@
+from festim import FluxBC, k_B
+import fenics as f
+import sympy as sp
+
+
+class MassFlux(FluxBC):
+    """
+    FluxBC subclass for advective mass flux
+    -D * grad(c) * n = h_mass * (c - c_ext)
+
+    Args:
+        h_mass (float or sp.Expr): mass transfer coefficient (m/s)
+        c_ext (float or sp.Expr): external concentration (1/m3)
+        surfaces (list or int): the surfaces of the BC
+    """
```

Comment:
Any chance we could add a reference in the docstrings?

Reply:
done!